src/backend/marsha/core/api/video.py

```diff
@@ -17,6 +17,7 @@
 from .. import defaults, forms, permissions, serializers, storage
 from ..models import LivePairing, Video
 from ..utils.api_utils import validate_signature
+from ..utils.channel_layers_utils import send_video
```

Comment:
IMO, I think `channel_layers_utils` should be part of websocket app.
And I feel that `send_video` isn't descriptive enough. Maybe `client_push_video` ?

Reply:
I don't know. This helper will be used in the core application only, it uses more code from the core application than the websocket one. 

Ok to rename this function.

Reply:
I renamed and finally moved it to the websocket application